components/inputs/input-textarea.js

```diff
@@ -86,7 +87,8 @@ class InputTextArea extends FocusMixin(LabelledMixin(FormElementMixin(SkeletonMi
 			 * Value of the input
 			 * @type {string}
 			 */
-			value: { type: String }
+			value: { type: String },
+			_hovered: { type: Boolean }
```

Comment:
We can now use reactive state properties for things we don't want to make public like this.

```suggestion
			_hovered: { state: true }
```